test/suites/tuned/profile.robot

```diff
@@ -0,0 +1,108 @@
+*** Settings ***
+Documentation       Tests for verification on MicroShift's Tuned profile
+
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/microshift-config.resource
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/ostree-health.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Test Cases ***
+X86 64 Should Run RT Kernel
+    [Documentation]    If system under test is x86_64, assert it's running RT kernel.
+
+    ${arch}=    Command Should Work    uname -m
+    IF    "${arch}" == "x86_64"
+        ${kernel}=    Command Should Work    sudo grubby --default-kernel
+        Should End With    ${kernel}    +rt
```

Comment:
Should Contain, maybe?

Reply:
Why?

Reply:
So that the check is more robust. I've never seen what it looks like on an RT kernel though.